railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -572,6 +572,10 @@ def run_after_bundle_callbacks
         @after_bundle_callbacks.each(&:call)
       end
 
+      def finish_message
+        say "Successfully created a new Rails app! Take a look at https://edgeguides.rubyonrails.org/getting_started.html for more info on next steps."
```

Comment:
```suggestion
        say "Successfully created a new Rails app! Take a look at https://guides.rubyonrails.org/getting_started.html for more info on next steps."
```

Reply:
`guides` will be guides for the current newest released version, whereas `edgeguides` follows the `main` branch. if a user has installed the gem from rubygems and run `rails new` they won't have all the code from the `main` branch.